modules/seqexec/server/src/main/scala/seqexec/server/tcs/Tcs.scala

```diff
@@ -35,16 +35,19 @@ final case class Tcs private (tcsController: TcsController,
 
   override val resource: Resource = Resource.TCS
 
+  private val usesAltair = gaos.flatMap(_.swap.toOption).isDefined
```

Comment:
Just out of curiosity, is there a reason you use `swap` here on the `Either` instead of just accessing `left`? This may be nitpicky, but it seems like it would be more clear to say:

```
gaos.exists(_.isLeft)
```

Reply:
Duh!